skyportal/utils/offset.py

```diff
@@ -1240,6 +1303,28 @@ def get_finding_chart(
             bbox=props,
         )
 
+    date_obs = hdr.get('DATE-OBS')
+    if not date_obs:
+        mjd_obs = hdr.get('MJD-OBS')
+        if mjd_obs:
+            date_obs = Time(f"{mjd_obs}", format='mjd').to_value(
+                'fits', subfmt='date_hms'
+            )
+
+    if date_obs:
+        ax.text(
+            0.95,
+            0.95,
+            f'image date {date_obs.split("T")[0]}',
+            horizontalalignment='right',
+            verticalalignment='center',
+            transform=ax.transAxes,
+            fontsize='small',
+            color="yellow",
+            alpha=0.5,
+            bbox=props,
+        )
+
```

Comment:
Should there be an `else` block here?

Reply:
No: `date_obs` will be `None` if it cannot be determined from the FITS header. In that case we will not be showing any date-of-image info on the finder. So no action is needed here.